dawn/test/unit-test/dawn/IIR/TestExtent.cpp

```diff
@@ -38,6 +38,21 @@ TEST(ExtentsTest, Construction) {
   Extents extents5{std::move(extents4)};
   EXPECT_EQ(extents5, expected);
 }
+TEST(ExtentsTest, EmptyConstruction) {
+  EXPECT_EQ(Extents(ast::cartesian), Extents());
+  EXPECT_NE(Extents(ast::cartesian, -1, 1, -1, 1, -1, 1), Extents());
+
+  EXPECT_EQ(Extents(), Extents());
+
+  Extents nullExtent;
+  Extents extents1{ast::cartesian, -1, 1, -1, 1, -1, 1};
+  extents1 = nullExtent;
+  EXPECT_EQ(extents1, nullExtent);
+
+  Extents extents2{ast::cartesian, -1, 1, -1, 1, -1, 1};
+  extents2 = std::move(nullExtent);
+  EXPECT_EQ(extents2, nullExtent);
```

Comment:
I'm not sure if this is what you want, as `nullExtent` has been `std::move`d

Reply:
Good point! Thanks.